src/Controller/AboutController.php

```diff
@@ -270,17 +271,22 @@ public function peopleAction(Request $request, string $type) : Response
                 $editorInChief = $leadership->filter(function (Person $person) {
                     return 'Editor-in-Chief' === $person->getTypeLabel();
                 });
-                $foundingEditorInChief = $leadership->filter(function (Person $person) {
-                    return 'Founding Editor-in-Chief' === $person->getTypeLabel();
-                });
+                // Load Founding Editor-in-Chief.
+                $foundingEditorInChief = $people->get('6d42f4fe')
```

Comment:
we don't need to put the id into configuration, but I'd at least extract it as a `const` in this class to make it very visible at the top

Reply:
`$foundingEditorInChief` is not plural but I think it's clear that it is of a list type rather than a single value (a list with 1 value, ok).